src/Field/Email.php

```diff
@@ -103,7 +103,6 @@ private function hasDNSRecord(string $domain, bool $isMX): bool
             return false;
         }
 
-        // dns_get_record can also return false
         $records = dns_get_record($normalizedDomain, ($isMX ? DNS_MX : DNS_A)) ?: [];
 
         return $records !== false && count($records) > 0;
```

Comment:
` ?: []` - with this you do not need `!== false`...

Reply:
oh yes, right :)